src/components/Label/Label.css

```diff
@@ -0,0 +1,3 @@
+.chi .chi-label__help .chi-button.-flat.-icon:not(.-light) .chi-icon {
+    color: #0262b9!important;

```

Comment:
Why is this necessary? Can not we use Chi color utility class here?
I believe, it's `-icon--primary`